components/depricated/RightSideBar/IndividualDate.tsx

```diff
@@ -1,12 +1,12 @@
 import React from "react";
 
-interface IndividualDate {
+interface IndividualDateCard {
   date: number;
   day: string;
   isTodayDate: boolean;
 }
 
-const IndividualDate = (props: IndividualDate) => {
+const IndividualDate = (props: IndividualDateCard) => {
   return (
     <div>
       <div
```

Comment:
Have you mentioned these changes in your issue #413  ?? If not please revert back the changes for all the files that were not mentioned in issue #413